app/index.js

```diff
@@ -83,31 +83,31 @@ module.exports = class GeneratorLicense extends Generator {
         name: 'name',
         message: 'What\'s your name:',
         default: this.options.name || this.gitc.user.name,
-        when: this.options.name == null
+        when: !this.options.name
```

Comment:
@ek9 allowing empty name or other values was first to prevent duplicating prompts when other generators prompts for a name but the user doesn't provide answer.